test/verify/check-packagekit

```diff
@@ -218,14 +233,32 @@ class TestUpdates(MachineCase):
         b.wait_in_text("#state", "Applying updates")
         b.wait_present("#app div.progress-bar")
 
-        # should have succeeded
-        b.wait_in_text("#state", "No updates pending")
-        # empty state visible in main area
-        b.wait_present(".container-fluid div.blank-slate-pf")
+        # should have succeeded and show restart
+        b.wait_present("#app .container-fluid h1")
+        b.wait_in_text("#app .container-fluid h1", "Restart Recommended")
+        b.wait_present("#app .container-fluid button.btn-default")
 
         # new versions are now installed
         m.execute("test -f /stamp-norefs-bin-2-1 && test -f /stamp-norefs-doc-2-1")
 
+        # do the reboot; this will disconnect the web UI
+        m.reset_reboot_flag()
+        b.click("#app .container-fluid button.btn-primary")
+        b.switch_to_top()
+        b.wait_visible(".curtains-ct")
```

Comment:
I was wondering about that too, but it seems to work fine. I cargo-culted this from check-ostree, check-shutdown-restart etc. It's of course entirely possible that they all do it wrong :-) Other tests use the standard timeout too. My feeling is that if our test VMs take more than 30s to shutdown, I'd rather fix that (they should take less than a second normally) instead of silently suffering from long test durations. That said, if you prefer I can bump the timeout.